recipes/wcurl/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "wcurl" %}
+{% set version = "2024.07.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/curl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 962bb72e36e6f6cedbd21c8ca3af50e7dadd587a49d2482ab3226e76cf6dcc97
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - curl >=7.83.0
+    - __unix  # [unix]
+    - __win   # [win]
+    - m2-bash  # [win]
+
+test:
+  commands:
+    - wcurl --help
+    - wcurl https://raw.githubusercontent.com/conda-forge/curl-feedstock/master/LICENSE.txt && test -f LICENSE.txt # [unix]
+    - wcurl https://raw.githubusercontent.com/conda-forge/curl-feedstock/master/LICENSE.txt && if not exist LICENSE.txt exit 1  # [win]
```

Each statement in `test/commands` is checked. So it won't run the next statement without the previous one passing

```suggestion
    - wcurl https://raw.githubusercontent.com/conda-forge/curl-feedstock/master/LICENSE.txt
    - test -f LICENSE.txt              # [unix]
    - if not exist LICENSE.txt exit 1  # [win]
```